extension/oauth2clientauthextension/testdata/test-cert.pem

```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Could you add a readme to this directory, with the commands used to provision this?

Reply:
Actually copied these from `configtls` module. I am actually planning to remove these tls tests as they are basically testing configtls  and are exactly the same

Reply:
@jpkrohling kindly let me know if reusing the test crts from another module is ok here (don't want to check in unecessary keys all over the place) or else I will generate new ones from cfssl instructions from your article (here)[https://medium.com/opentelemetry/securing-your-opentelemetry-collector-1a4f9fa5bd6f]